.travis.yml

```diff
@@ -40,7 +40,7 @@ before_script:
   - ./tests/travis_${TRANSPORT}.sh
 
 script: 
-  - phpunit -c tests/phpunit_${TRANSPORT}.xml.dist
+  - ./vendor/bin/phpunit -c tests/phpunit_${TRANSPORT}.xml.dist
```

Comment:
can you fix the build in the branch of #709 please? it should also be in the maintenance branch and has nothing to do with this feature.